recipes/r-ngsplot/scripts/activate.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+export NGSPLOT=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
```

Comment:
Oh, and using `$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM` should not work here, I believe.
Those variable are only defined during the build/test phase or during `post-link` (and others), but not when activating an environment. If the tests passed for this, that's only because they were set in that case.
You'll want to create/modify the activation script in `build.sh`.